wapi/curves.py

```diff
@@ -1019,3 +1164,166 @@ def get_latest(self, tags=None, issue_date_from=None, issue_date_to=None, issue_
         if result is None:
             return result
         return util.TS(input_dict=result, curve_type=util.TAGGED_INSTANCES)
+
+
+    def get_relative(self, data_offset, data_max_length=None, tag=None, issue_date_from=None, issue_date_to=None,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method "get_relative" has 18 parameters, which is greater than the 15 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dev.wattsight.com/coding_rules#rule_key=python%3AS107)